package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Nov 26 12:22:50 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Abort the execution when the module run without enough
+  permissions (related to bsc#1137688).
```

Comment:
It's actually hard to see the relationship with bsc#1137688, but I trust you ;-)

Reply:
Yes, it's hard also to me. It's what I said in [a comment](https://trello.com/c/nhBiaGEa/1103-2-yast2-systemsettings-crashes-with-internal-error#comment-5ddbaa0431bc5f8fe0393e55) in the Trello card :man_shrugging: